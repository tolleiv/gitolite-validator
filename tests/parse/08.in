@open = git gitolite foss/..* [...]

repo @all
    -   =   gitweb daemon
    option deny-rules = 1

repo @open
    R   =   gitweb daemon
    config hooks.emailprefix    = '[%GL_REPO]'
    option deny-rules = 0
