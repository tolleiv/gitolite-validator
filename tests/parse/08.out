root:
  group:
    name: open
    members:
      git
      gitolite
      foss/..*
      [...]
  repo:
    paths:
      @all
    rules:
      rule:
        permission: -
        members:
          gitweb
          daemon
    options:
        deny-rules = 1
  repo:
    paths:
      @open
    rules:
      rule:
        permission: R
        members:
          gitweb
          daemon
    configs:
        hooks.emailprefix = '[%GL_REPO]'
    options:
        deny-rules = 0
